src/classes/RD_Constants.cls-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>47.0</apiVersion>
```

Comment:
Version 46.0?

Reply:
Since v48 is going live next week, we're fine with using v47 on new classes.